tox.ini

```diff
@@ -72,7 +72,7 @@ commands =
 setenv = 
     DJANGO_SETTINGS_MODULE = test_settings
 deps = 
-    -r{toxinidir}/requirements/dev.txt
+    -r{toxinidir}/requirements/test.txt
```

Comment:
Can you please explain what this does?

Reply:
`pii annotations` were erroring out with test.txt requirements, so it was replaced with `dev.txt` temporarily in this PR. Now switch it back to use the `test.txt` file to run `pii annotations`.